doc/examples/tissue_classification.py

```diff
@@ -32,7 +33,7 @@
 print('t1.shape (%d, %d, %d)' % t1.shape)
 
 """
-We have fetched the T1 volume. Now we will look at the axial and the coronal
+We have fetched T1 volume. Now we will look at the axial and coronal
```

Comment:
I'd dare to say that the article is necessary/correct.

Reply:
Appreciate for reviewing @jhlegarreta but I still think the change is correct and I will for any maintainer to review it.

Reply:
The second `the` is not absolutely necessary, but the first one (before `T1`) should be kept.

Reply:
Can you update @ShrishtiHore? Thank you